crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -404,26 +333,27 @@ impl VM {
         current_call_frame: &mut CallFrame,
     ) -> Result<OpcodeSuccess, VMError> {
         let value_in_wei_to_send = current_call_frame.stack.pop()?;
-        let code_offset_in_memory: usize = current_call_frame
-            .stack
-            .pop()?
-            .try_into()
-            .map_err(|_err| VMError::VeryLargeNumber)?;
+        let code_offset_in_memory = current_call_frame.stack.pop()?;
         let code_size_in_memory: usize = current_call_frame
             .stack
             .pop()?
             .try_into()
             .map_err(|_err| VMError::VeryLargeNumber)?;
         let salt = current_call_frame.stack.pop()?;
 
+        let new_size: usize = code_offset_in_memory
```

Comment:
ditto

Reply:
Done in [7d306fa](https://github.com/lambdaclass/ethrex/pull/1458/commits/7d306fae02205b1461716c08e2c12338a7176741)